api/v1beta1/flowcollector_types.go

```diff
@@ -226,27 +232,17 @@ type FlowCollectorEBPF struct {
 	// +optional
 	Debug DebugConfig `json:"debug,omitempty"`
 
-	// Enable the Packets drop flows logging feature. This feature requires mounting
+	// List of additional features to enable. They are all disabled by default. Enabling additional features may have performance impacts. Possible values are:<br>
+	// - `PKT_DROP`: enable the packets drop flows logging feature. This feature requires mounting
 	// the kernel debug filesystem, so the eBPF pod has to run as privileged.
-	// If the spec.agent.eBPF.privileged parameter is not set, an error is reported.
-	//+kubebuilder:default:=DISABLED
-	//+optional
-	PktDrop FeatureConfigType `json:"enablePktDrop,omitempty"`
-
-	// Enable the DNS tracking feature. This feature requires mounting
+	// If the `spec.agent.eBPF.privileged` parameter is not set, an error is reported.<br>
+	// - `DNS_TRACKING`: enable the DNS tracking feature. This feature requires mounting
 	// the kernel debug filesystem hence the eBPF pod has to run as privileged.
-	// If the spec.agent.eBPF.privileged parameter is not set, an error is reported.
-	//+kubebuilder:default:=DISABLED
-	//+optional
-	DNSTracking FeatureConfigType `json:"enableDNSTracking,omitempty"`
-
-	// `enableFlowRtt` allows enabling FlowRTT calculations in the ebpf agent.
-	// Currently only TCP handshake based RTT is calculated per flow.
-	// This feature is optionally enabled and is disabled by default.
-	// This feature needs both INGRESS and EGRESS direction flow capture and will be disabled if both are not enabled.
-	//+kubebuilder:default:=DISABLED
-	//+optional
-	FlowRTT FeatureConfigType `json:"enableFlowRtt,omitempty"`
+	// If the `spec.agent.eBPF.privileged` parameter is not set, an error is reported.<br>
+	// - `FLOW_RTT` <i>Unsupported (*)</i>: allows enabling flow latency (RTT) calculations in the eBPF agent during TCP handshakes.
+	// This feature needs both INGRESS and EGRESS direction flow capture and will be disabled if they are not both enabled.<br>
+	// +optional
```

Comment:
should we add
```sh
// +kubebuilder:validation:Required
// +kubebuilder:validation:MinItems:=1
```

Reply:
yeah I don't even think an empty array initialization is needed, we don't do that in other places (e.g. exporters list)